src/ansible_navigator/actions/config.py

```diff
@@ -321,7 +321,7 @@ def _parse_and_merge(self, list_output, dump_output) -> None:
                         parsed[variable]["via"] = source
                     parsed[variable]["current"] = current
                     parsed[variable]["__current_value"] = extracted.groupdict()["current"]
-                except KeyError as exc:  # noqa: F841
+                except KeyError as _exc:  # noqa: F841
                     self._logger.error("variable '%s' not found in list output")
```

Comment:
have you considered just adding `exc_info=exc` or dropping the `as` part?

Reply:
ha! I didn't even think of that, I'll do another PR and look for others too.